.github/workflows/issue_slack.yml

```
@@ -0,0 +1,79 @@
+
+jobs:
+  send-messages:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run github variable preprocess
+        uses: jannekem/run-python-script-action@v1
+        id: preprocess
+        with:
+          created_time: ${{ github.event.issue.created_at }}
+          script: |
+        from datetime import datetime, timedelta, timezone
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)